rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/does_hvac_system_serve_single_zone.py

```diff
@@ -0,0 +1,17 @@
+def does_hvac_system_serve_single_zone(rmi_b, zone_id_list):
+    """Returns TRUE if the HVAC system serves a single zone. Returns FALSE if the HVAC system serves multiple zones.
```

Comment:
Shouldn't this function take _hvac_b_id_ system id as an argument instead of _zone_id_list_?

Reply:
Yea, I've been tracing the code, the higher level functions (e.g. [is_baseline_system_7](https://github.com/pnnl/ruleset-checking-tool/pull/553/files) seems to have `zone_id_list` as one of the arguments. It maybe logical to use the `zone_id_list` for this purpose because it is a lot straightforward.

Reply:
@weilixu These two functions should help with your understanding of how these work together. [get_baseline_system_type](https://github.com/pnnl/ruleset-checking-tool/pull/223) and [get_dict_of_zones_and_terminal_units_served_by_hvac_sys](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fpnnl%2Fruleset-checking-tool%2Fpull%2F523&data=05%7C01%7CSupriya.Goel%40pnnl.gov%7Cdc3f58f61aa6401edeaf08da8b64923e%7Cd6faa5f90ae240338c0130048a38deeb%7C0%7C0%7C637975561372861891%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=XTsCgXHlpqnGIwVHg8kHq96zagsmHGxlZIqmEjuwW%2FI%3D&reserved=0)()

Reply:
Ok. I now understand why _zone_id_list_ is passed instead of _hvac_b_id_. So my question is resolved. Thank's for pointing me to _get_baseline_system_type_ @claperle .